l10n_br_base/models/res_partner.py

```diff
@@ -20,6 +20,16 @@ class Partner(models.Model):
     _name = "res.partner"
     _inherit = [_name, "l10n_br_base.party.mixin"]
 
+    def _inverse_street_data(self):
+        """update self.street based on street_name, street_number and street_number2"""
+        for partner in self:
```

Comment:
A gente ta se esforçando para que os módulos centrais da localização funcionam com bancos muti-pais, nisso
tem que filtrar apenas os registros pros quais partner.country_id.code.upper() == "BR" e senão chamar super.

Eu tb acho melhor falar no docstring o que a gente faz no override do que fazer o papagaio do docstring original.
Poderia ser algo do tipo:
""'"In Brazil the address format is street_name, street_number (comma instead of space)"""

Reply:
Solved